frontend/public/components/utils/_resource-log.scss

```diff
@@ -20,3 +20,7 @@
   display: grid;
   grid-template-rows: max-content;
 }
+
+.pf-c-log-viewer.pf-m-nowrap {
```

Comment:
I'd be inclined to move this to https://github.com/openshift/console/blob/master/frontend/public/style/_overrides.scss (specifically [here](https://github.com/openshift/console/blob/134c329b63aa2a63d25879832033f4055561d7fc/frontend/public/style/_overrides.scss#L159)) given the selector is specific to PatternFly and not anything in our specific implementation.

Reply:
Moving to _overrides.scss, since this covers the Log Viewer component in both resource-log.tsx and NodeLogs.tsx